app/services/invoices/calculate_fees_service.rb

```diff
@@ -223,6 +223,8 @@ def should_create_subscription_fee?(subscription, boundaries)
       return false unless should_create_yearly_subscription_fee?(subscription)
       return false unless should_create_semiannual_subscription_fee?(subscription)
       return false if in_trial_period_not_ending_today?(subscription, boundaries.timestamp)
+      # now we have a case, where we bill a subscription on the first day, but it's not a pay_in_advance plan - it includes pay_in_advance fixed_charges
+      return false if billing_advance_fixed_charges_on_first_invoice?(subscription)
```

Comment:
if plan has no fixed charges. this logic will not be executed. In the next PRs it will be changed